lineman/app/components/navbar/navbar.haml

```diff
@@ -7,9 +7,9 @@
     .lmo-navbar__item{ng-class: '{"lmo-navbar__item--selected": selected == "inboxPage"}'}
       %a.btn.lmo-navbar__btn.lmo-navbar__btn-icon.has-badge{href: '/inbox', title: "{{ 'navbar.unread_title' | translate }}", tabindex: 2}
         %i.fa.fa-lg.fa-inbox>
+          %span.badge{ng-show: 'unreadThreadCount() > 0'}
```

Comment:
Hm, why this move? 

Not sure it feels right to nest a span inside an <i> (isn't it funny how fontawesome has basically gotten 100% squatters right on the italic html tag?), and it feels like this would behave the same without it, since it'll be positioned against the relative <a> above it?